src/main/java/io/stargate/sgv2/jsonapi/service/shredding/Shredder.java

```diff
@@ -70,8 +85,13 @@ public WritableShreddedDocument shred(JsonNode doc, UUID txId) {
     // Need to re-serialize document now that _id is normalized.
     // Also gets rid of pretty-printing (if any) and unifies escaping.
     try {
+      Timer.Sample sample = Timer.start(meterRegistry);
       // Important! Must use configured ObjectMapper for serialization, NOT JsonNode.toString()
       docJson = objectMapper.writeValueAsString(docWithId);
+      Tag commandTag = Tag.of(jsonApiMetricsConfig.command(), commandName);
+      Tag serializationTag = Tag.of(jsonApiMetricsConfig.serializationJson(), "true");
+      Tags tags = Tags.of(commandTag, serializationTag);
+      sample.stop(meterRegistry.timer(jsonApiMetricsConfig.serializationMetricsName(), tags));
```

Comment:
I think code for timing should probably start outside of `try` block, and `sample.stop()` called in `finally` block.
This way we would get timings for failing cases too?

I guess code here will get refactored if there is wrapper/callback.